dataland-frontend/src/components/general/AlignedActivitiesDataTable.vue

```diff
@@ -308,18 +334,20 @@ function createRevenueGroupData(activity: EuTaxonomyAlignedActivity): ActivityFi
 function createActivityGroupData<T>(
   activityName: string,
   groupName: string,
-  fields: { [key: string]: T } | undefined,
+  fields: { [key: string]: T | null | undefined } | undefined,
```

Comment:
Why is this necessary?

Reply:
I removed it because it is indeed not necessary.

Reply:
ok